google-client-loader.html

```diff
@@ -140,6 +140,9 @@
       ready: function() {
         this._loader = document.createElement('google-js-api');
         this.listen(this._loader, 'js-api-load', '_loadClient');
+        if (this._loader.libraryLoaded) {
+            this._loader.fire('js-api-load');
```

Comment:
Actually, the listening for the js-api-load event is not needed if the library is already loaded. This can be replaced by:
```
if (this._loader.libraryLoaded) {
  this._loadClient();
} else {
  this.listen(this._loader, 'js-api-load', '_loadClient');
}
```